pkg/alicloud/errors/codes.go

```diff
@@ -27,6 +27,8 @@ const (
 	InvalidVpcZoneNotSupported = "InvalidVpcZone.NotSupported"
 	// InvalidZoneIDNotSupportShareEncryptedImage: Creating instances by shared encrypted images is not supported in this zone.
 	InvalidZoneIDNotSupportShareEncryptedImage = "InvalidZoneId.NotSupportShareEncryptedImage"
+	// InvalidResourceType.NotSupported: The specified resource does not exist. Please try other zone or configuration.
+	InvalidResourceTypeNotSupported = "InvalidResourceType.NotSupported"
 	// InvalidInstanceType.ZoneNotSupported: The specified zone does not support this instancetype.
```

Comment:
I know this is not a comment introduced by you, but could you also correct this godoc to correctly have the variable name?
```suggestion
	// InvalidInstanceTypeZoneNotSupported: The specified zone does not support this instancetype.
```